zap/src/main/java/org/parosproxy/paros/db/paros/ParosTableAlertTag.java

```diff
@@ -0,0 +1,195 @@
+package org.parosproxy.paros.db.paros;
```

Comment:
Needs header. Could be called Zap___.java ? (@psiinon @thc202)

Reply:
It's "Paros" because it's the legacy DB (vs Sql ones).

Reply:
Oh okay